unit_tests/constants.py

```diff
@@ -0,0 +1,25 @@
+test_simulation = {
+    "username": "user@test.com",
+    "name": "AggregatorTestSuite",
+    "uuid": "some_random_uuid",
+    "domain_name": "https://cool.webpage.com",
+    "websockets_domain_name": "wss://cool.webpage.com/external-ws",
+}
+
+test_agg = {
```

Comment:
I would vote for having two constants here. If I read `test_agg` in the other module, I would expect an Aggregator instance. My proposal:
```python
TEST_AGGREGATOR_NAME = "TestAggr"
TEST_AGGREGATOR_UUID = str(uuid.uuid4())
```